Parsers/ASimFileEvent/Parsers/vimFileEventVMwareCarbonBlackCloud.yaml

```diff
@@ -0,0 +1,150 @@
+                          temp_action has "create", "FileCreated",
+                          ""
+                      )
+      | extend
+          EventStartTime = todatetime(split(createTime_s, '+')[0]),
+          TargetFilePathType = case(
+                          device_os_s == "WINDOWS" and filemod_name_s startswith "\\", "Windows Share",
+                          device_os_s == "WINDOWS", "Windows Local",
+                          device_os_s in ("MAC", "LINUX"), "Unix",
+                          ""
+                      ),
+          ActingProcessId = tostring(toint(process_pid_d)),
+          TargetFileName = tostring(split(filemod_name_s, '\\')[-1]),
+          AdditionalFields = bag_pack(
+                        "org_key", org_key_s,
+                        "process_publisher", process_publisher_s,
+                        "process_reputation", process_reputation_s,
+                        "process_guid", process_guid_s
+                    )
+      | invoke _ASIM_ResolveDvcFQDN('device_name_s')
+      | project-rename
+          ActorUsername = process_username_s,
+          DvcIpAddr = device_external_ip_s,
+          EventUid = _ItemId,
+          DvcScope = device_group_s,
+          ActingProcessCommandLine = process_cmdline_s,
+          ActingProcessName = process_path_s,
+          DvcId = device_id_s,
+          DvcOriginalAction = sensor_action_s,
+          DvcOs = device_os_s,
+          EventMessage = event_description_s,
+          EventOriginalType = action_s,
+          EventOriginalUid = event_id_g,
+          EventOwner = event_origin_s,
+          TargetFilePath = filemod_name_s
+      | extend 
+          EventProduct = "Carbon Black Cloud",
+          EventSchema = "FileEvent",
+          EventSchemaVersion = "0.2.1",
+          EventVendor = "VMware",
+          EventCount = int(1)
+      | extend
+          EventEndTime = EventStartTime,
+          Dvc = coalesce(DvcId, DvcIpAddr, DvcHostname),
+          FileName = TargetFileName,
+          FilePath = TargetFilePath,
+          Process = ActingProcessName,
+          User = ActorUsername,
+          Hash = coalesce(TargetFileSHA256, TargetFileSHA256)
+      | extend
+          ActorUsernameType = _ASIM_GetUsernameType(ActorUsername),
+          ActorUserType = _ASIM_GetUserType(ActorUsername, ""),
+          DvcIdType = iff(isnotempty(DvcId), "Other", ""),
+          HashType = case(
+                isnotempty(TargetFileSHA256),
+                "TargetFileSHA256",
+                isnotempty(TargetFileMD5),
+                "TargetFileMD5",
+                ""
+            )
+      | project-away
+          *_s,
+          *_d,
+          *_g,
+          *_b,
+          _ResourceId,
+          Computer,
+          MG,
+          ManagementGroupName,
+          RawData,
+          SourceSystem,
+          TenantId,
+          temp_action
+  };
+  parser(disabled=disabled)
```

Comment:
can you list filters we can use with file? we can discuss more over call.

Reply:
As discussed, we have added filters for vim parser.